build.gradle

```diff
@@ -213,8 +213,8 @@ def fixAscNames = { name ->
 }
 
 compileJava {
-    sourceCompatibility = '17'
-    targetCompatibility = '17'
+    sourceCompatibility = '11'
```

Comment:
```suggestion
    sourceCompatibility = '17'
```

as we're compiling with Java 17 we can use the most recent language features that don't require additional capabilities in the byte code.
https://stackoverflow.com/questions/16654951/gradle-sourcecompatibility-vs-targetcompatibility